LoopKit/InsulinKit/BolusRecommendation.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  BolusRecommendation.swift
+//  LoopKit
+//
+//  Created by Pete Schwamb on 1/2/17.
+//  Copyright © 2017 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+public enum BolusRecommendationNotice {
+    case glucoseBelowSuspendThreshold(minGlucose: GlucoseValue)
+    case currentGlucoseBelowTarget(glucose: GlucoseValue)
+    case predictedGlucoseBelowTarget(minGlucose: GlucoseValue)
```

Comment:
[nit] 
- should `glucoseBelowSuspendThreshold(minGlucose: GlucoseValue)` be  `glucoseBelowSuspendThreshold(suspendThrehold: GlucoseValue)`
- should 'currentGlucoseBelowTarget(glucose: GlucoseValue)` be currentGlucoseBelowTarget(targetGlucose: GlucoseValue)`
- should `predictedGlucoseBelowTarget(minGlucose: GlucoseValue)` be `predictedGlucoseBelowTarget(targetGlucose: GlucoseValue)`

I guess I'm saying, I don't understand what `minGlucose` is and it makes me think it is the referenced value instead of the current value.

Reply:
Yes, that makes sense to me. However, I just copied this file from `Loop` into `LoopKit` to make it accessible to the services, so I don't think it is appropriate for me to change it with this PR.